samples/WebFormsSample/TestUserControl.ascx

```diff
@@ -0,0 +1,31 @@
+<%@ Control Language="C#" AutoEventWireup="true" CodeBehind="TestUserControl.ascx.cs" Inherits="SystemWebUISample.TestUserControl" %>
+<%@ Register Src="~/InsideTestUserControl.ascx" TagPrefix="ucInside" TagName="InnerTestUserControl" %>
+
+<div id="socialLoginList">
```

Comment:
btw can you simplify this control? we don't need it to do so much when all we want is a basic user control